drop-in/src/main/AndroidManifest.xml

```diff
@@ -19,9 +20,16 @@
 
                 <data
                     android:host="${applicationId}"
+                    android:path=""
+                    android:scheme="${checkoutRedirectScheme}"
+                    tools:ignore="AppLinkUrlError" />
```

Comment:
Out of curiosity, why do we 2 data tags and the tools ignore?

Reply:
The first one with `""` is to handle deep links like `scheme://host`
The first second one with `"/"` is to handle deep links like `scheme://host/`
It's just defensive, because some libraries/people always suffix URLs with a slash